components/x-audio/src/redux/index.jsx

```diff
@@ -30,12 +32,20 @@ export default function connectPlayer (Player) {
 			notifiersProxy.set(props.notifiers);
 			this.unsubscribe = store.subscribe(this.storeUpdated.bind(this));
 			this.onCloseClick = this.onCloseClick.bind(this);
+			this.setExpandedPlayerRef = element => {
+				this.expandedPlayerRef = element;
+			};
+			this.hasSetSwipeAnimation = false;
 			this.state = initialState;
 		}
 
 		componentDidMount() {
 			const { playing, url, trackingContext } = this.props;
 			playerActions.loadMedia({ url, trackingContext, autoplay: playing });
+
+			if (this.expandedPlayerRef && !this.hasSetSwipeAnimation) {
+				this.setSwipeDownAnimation();
```

Comment:
I wonder if this would be clearer named as `listenForSwipeDown`?

Reply:
👍